style.css

```diff
@@ -0,0 +1,167 @@
+body{
+  background-color: #DFE5DC;
+  color: #234d2c;
+  font-family: sans-serif, serif;
+}
+
+.game{
+  display: inline-flex;
+}
+
+.game > div{
+  margin-left: 20px;
+}
+
+.game-play .pay-table{
+  display: flex;
+  vertical-align: top;
+  flex-wrap: wrap;
+  /* margin: 20px 30px; */
+}
+
+.game-play{
+  text-align: center;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid olive;
+  border-radius: 9px;
+}
+
+.inputs{
+  margin-top: 10px;
+}
+.bet-amount{
+  border-radius: 3px;
+  border: 1px solid olive;
+}
+
+ 
+.pay-table{
+  justify-content: flex-start;
+} 
+
+table, th, td{
+  border:1px solid olive;
+  border-collapse: collapse;
+  padding: 10px;
+  font-size: 90%;
+}
+
+.button-common{
+  background-color: #3B6855;
+  color: white;
+  text-align: center;
+  border-radius: 9px;
+  margin: 10px;
+  padding: 7px 7px ;
+}
+
+.stand-alone-button{
+  font-size: 1.1em;
+  border-radius: 6px;
+  padding: 10px 15px ;
+}
+
+
+.single-card{
+  display: inline-block;
+  vertical-align: top;
+  margin: 10px;
+  width: 100px;
+  height: 125px;
+  background-color: #cee0c4;  
+  border-radius: 5px;
+  text-align:center;
+  color:black;
+}
+.red{
+    color:red;
+  }
+
+.suit{
+  margin: 5px;
+  font-size:20px;
+}
+.name{
+  margin: 5px;
+  font-size:24px;
+  font-weight:bold;
+  font-family:sans-serif;
+}
+
+.hold-status{
+  font-size: 9px;
+}
+
+.default-fill{
+  width: 45px;
+  margin: 10%;
+  overflow: hidden;
+}
+
+.reduced-card{
+  width: 25px;
+  height: 35px;
+  margin: 5px;
+  font-size:10px;
+  font-weight:bold;
+  font-family:sans-serif;
+}
+
+
+.reduced-size{
+  margin: 5px;
+  font-size:10px;
+  font-weight:bold;
+  font-family:sans-serif;
+}
+
+.status{
+  color: #0c800c;
+  font-weight: lighter;
+  font-style: italic;
+  font-size: 90%;
+  margin: 10px 10px;
+  line-height: 1.5em;
+}
+
+.result{
+  margin-top: 30px;
+}
+.score-card-container{
+  margin: 20px;
+}
+
+.tooltip .tooltip-text {
+  visibility: hidden;
```

Comment:
Great work experimenting with CSS. Consider adding more comments to your CSS file and grouping properties together to more clearly communicate what you are trying to achieve.